src/app/stelliance-connect-widget-apps/stelliance-connect-widget-apps.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { StellianceConnectWidgetConfig } from '../stelliance-connect-widget/stelliance-connect-widget-config.model';
+
+@Component({
+  selector: 'app-stelliance-connect-widget-apps',
+  templateUrl: './stelliance-connect-widget-apps.component.html',
+  styleUrls: ['./stelliance-connect-widget-apps.component.css'],
+})
+export class StellianceConnectWidgetAppsComponent {
+  @Input() apps?: StellianceConnectWidgetConfig[];
```

Comment:
```suggestion
  @Input() apps: StellianceConnectWidgetConfig[];
```
Pas besoin du `?`, l'input `apps` est obligatoire, pareil pour les autres input